Src/Newtonsoft.Json/Bson/BsonWriter.cs

```diff
@@ -72,15 +72,15 @@ public BsonWriter(Stream stream)
         /// <summary>
         /// Initializes a new instance of the <see cref="BsonWriter"/> class.
         /// </summary>
-        /// <param name="writer">The writer.</param>
+        /// <param name="writer">The <see cref="BinaryWriter"/> to write to.</param>
         public BsonWriter(BinaryWriter writer)
         {
             ValidationUtils.ArgumentNotNull(writer, nameof(writer));
             _writer = new BsonBinaryWriter(writer);
         }
 
         /// <summary>
-        /// Flushes whatever is in the buffer to the underlying streams and also flushes the underlying stream.
+        /// Flushes whatever is in the buffer to the underlying <see cref="BinaryWriter"/> or <see cref="Stream"/> and also flushes the underlying writer or stream.
```

Comment:
A BinaryWriter can only wrap a stream so I don't think it needs to be mentioned. Repeat in other comments.